### PR TITLE
📆 Add Simon's keynote

### PR DESCRIPTION
**Pull request by drewbrew**
Changes proposed in this PR:

- Add opening and closing remarks for Wednesday
- Add Simon's keynote to schedule
- Add blog post announcing Simon's keynote

Per Simon,

> I still haven't had a chance to snap a picture so I think we'll do without :)

## Screenshots
image

image

image

image

### CODE REVIEW

**Review by jefftriplett — APPROVED** [review]

**Comment by logankilpatrick** [comment]
Quick question, can we put the verbose name for ORM? It might make it more clear, I use Django but am not sure I know what ORM is

**Comment by drewbrew** [comment]
> Recent releases of Django introduced ORM (object-relational mapping) features

I added it in the first line of the talk description. I'm happy to add it to the title and/or blog post if you think it's worthwhile.

**Comment by logankilpatrick** [comment]
I would prefer that @drewbrew but that's my own preference, maybe everyone else know what ORM means if they are at DjangoCon

**Comment by drewbrew** [comment]
Good by me. I'll add it in the morning. 

Also yay input lag on accidentally closing this.

**Review by jefftriplett — APPROVED** [review]

File: _posts/2022-08-08-simon-charette.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+author: Drew Winstel
+category: Program
+date: 2022-08-08 16:00:00-07:00
+layout: post
+title: 'Announcing Our Keynotes: Simon Charette'
+---
+
+We're honored to announce our third keynote speaker for DjangoCon US 2022: Simon Charette!
+
+Simon has been contributing to Django for the past ten years, mainly to the ORM and the migration framework. Through this work, he developed an interest in RDBMS/SQL and has been focusing on making Django work great with them in the context of large Python applications.
```
Comment (drewbrew) [inline _posts/2022-08-08-simon-charette.md]:
```suggestion
Simon has been contributing to Django for the past ten years, mainly to the ORM (object-relational mapping) and the migration framework. Through this work, he developed an interest in RDBMS/SQL and has been focusing on making Django work great with them in the context of large Python applications.
```

File: _schedule/talks/2022-10-19-09-45-t0-keynote-state-of-orm.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+end_date: 2022-10-19 10:30:00-07:00
+layout: session-details
+link: null
+permalink: /talks/keynote-state-of-orm
+presenter_slugs:
+- simon-charette
+room: Salon A-E
+sitemap: true
+talk_slot: full
+title: 'Keynote: State of the ORM'
```
Comment (drewbrew) [inline _schedule/talks/2022-10-19-09-45-t0-keynote-state-of-orm.md]:
```suggestion
title: 'Keynote: State of the Object-Relational Mapping (ORM)'
```

**Pull request merged**